src/iohandler/curl_io_handler.cc

```diff
@@ -35,15 +35,16 @@
 #ifdef HAVE_CURL
 #include "curl_io_handler.h" // API
 
+#include <utility>
```

Comment:
Standard includes should go after the other to avoid implicit availablity of stuff